code/modules/mob/living/simple_animal/bot/bot.dm

```diff
@@ -393,21 +429,10 @@
 /mob/living/simple_animal/bot/attackby(obj/item/attacking_item, mob/living/user, params)
 	if(attacking_item.GetID())
 		unlock_with_id(user)
-	else if(istype(attacking_item, /obj/item/pai_card))
-		insertpai(user, attacking_item)
-	else if(attacking_item.tool_behaviour == TOOL_HEMOSTAT && paicard)
-		if(bot_cover_flags & BOT_COVER_OPEN)
-			to_chat(user, span_warning("Close the access panel before manipulating the personality slot!"))
-		else
-			to_chat(user, span_notice("You attempt to pull [paicard] free..."))
-			if(do_after(user, 30, target = src))
-				if (paicard)
-					user.visible_message(span_notice("[user] uses [attacking_item] to pull [paicard] out of [initial(src.name)]!"),span_notice("You pull [paicard] out of [initial(src.name)] with [attacking_item]."))
-					ejectpai(user)
-	else
-		if(attacking_item.force) //if force is non-zero
-			do_sparks(5, TRUE, src)
-		..()
+		return
+	if(attacking_item.force) //if force is non-zero
+		do_sparks(5, TRUE, src)
+	return ..()
```

Comment:
If you may please, move this to `attacked_by` (`.` being truthy means successful attack)